app/domain/authentication/authn_oidc/authenticate_id_token/authenticate.rb

```diff
@@ -72,7 +72,7 @@ def conjur_username
           id_token_username_field = oidc_secrets["id-token-user-property"]
 
           conjur_username = id_token_attributes[id_token_username_field]
-          raise IdTokenFieldNotFound, id_token_username_field unless conjur_username.present?
+          raise IdTokenFieldNotFound, id_token_username_field unless conjur_username
```

Comment:
why was this changed? i had the `present?` to verify that not only the field exists but that it has a value. is this not needed?

Reply:
I think this was correct with `present?`. Also interested to know why it changed.

Reply:
You are right. The only issue is that the error not fully reflect the error. I will modify the name and the log to IdTokenFieldNotFoundOrEmpty